gems/pending/appliance_console/internal_database_configuration.rb

```diff
@@ -46,7 +46,7 @@ def activate
 
     def ask_questions
       choose_disk
-      self.run_as_evm_server = ask_yn?("Do you also want to use this server as an application server")
+      self.run_as_evm_server = ask_yn?("Is this a standard install? (Enter N if it will be a database ONLY)")
```

Comment:
I don't particularly like the wording "standard install". That may be clear for people that use the product already, but would be confusing for anyone else as this doesn't explain what "standard" is.

I like the idea of reversing the question to ask if the appliance should be a "Database only appliance". Is that more clear than "application server"?